.nvmrc

```diff
@@ -1 +1 @@
-18.12
+lts/*
```

Comment:
I’m guessing you checked, but no danger of this putting us out of step with Netlify with respect to Node versions?